packages/teams-js/src/public/pages.ts

```diff
@@ -690,4 +690,55 @@ export namespace pages {
       return runtime.supports.pages ? (runtime.supports.pages.appButton ? true : false) : false;
     }
   }
+
+  /**
+   * Parameters for the NavigateToSelf API
+   */
+  export interface NavigateToSelfParams {
+    /**
+     * Developer-defined ID of the Page to navigate to within the application (Formerly EntityID)
+     */
+    pageId: string;
+
+    /**
+     * Optional developer-defined ID describing the content to navigate to within the Page. This will be passed
+     * back to the application via the Context object.
```

Comment:
We should use @linkcode to link back to the correct part of context. In this case I think it would be app.context.page.subPageId